ramp-database/ramp_database/tools/user.py

```diff
@@ -104,6 +104,21 @@ def add_user(session, name, password, lastname, firstname, email,
     return user
 
 
+def delete_user(session, name):
+    """Delete a user from the database.
+
+    Parameters
+    ----------
+    session : :class:`sqlalchemy.orm.Session`
+        The session to directly perform the operation on the database.
+    name : str
+        The name of the user.
+    """
+    user = session.query(User).filter(User.name == name).one()
+    session.delete(user)
```

Comment:
Should we delete also teams he belonged to and the submissions ?

Reply:
`session.delete` will make a cascade delete where there is a foreign key. Team should have a foreign key there. What could be added is that it really happen in the team table.

Reply:
I pushed a quick additional check: https://github.com/paris-saclay-cds/ramp-board/commit/c73e2fa3d7f6bf15a88e7b72dc45aeb8e2cac69c